.gitignore

```diff
@@ -1,2 +1,3 @@
 lib
 node_modules
+yarn.lock
```

Comment:
I don't think that this is as open and shut as that blog post makes it out to be.

When there's a choice between taking pain upon ourselves vs pushing pain onto our users, I think the ratio has to be pretty large for it to be net positive for us to push the pain down onto users.

Having travis notify us about upstream changes that break us is pretty nice as well, as it happens right when someone is actually working on the repo, and a failing test is probably going to be easier to debug than an issue as reported by a user.